tools/generator/src/DTO/Project.swift

```diff
@@ -6,7 +6,7 @@ struct Project: Equatable, Decodable {
     let buildSettings: [String: BuildSetting]
     var targets: [TargetID: Target]
     let targetMerges: [TargetID: Set<TargetID>]
-    let targetHosts: [TargetID: Set<TargetID>]
+    let targetHosts: [TargetID: [TargetID]]
```

Comment:
Curious. Why are these no longer unique?

Reply:
They were always unique by construct. I needed them to be ordered as well.